docs/examples/fork-process-model/flask-gunicorn/requirements.txt

```diff
@@ -12,7 +12,7 @@ opentelemetry-instrumentation==0.41b0
 opentelemetry-instrumentation-flask==0.41b0
 opentelemetry-instrumentation-wsgi==0.41b0
 opentelemetry-sdk==1.20.0
-protobuf==3.19.5
```

Comment:
Better to use only one version of `protobuf` `3.x.y` and `4.x.y` in the whole repo in my opinion.